packages/docs/docs/components/code-block.mdx

```diff
@@ -109,20 +132,24 @@ yield view.add(
   <CodeBlock
     code={`var myBool;`}
     selection={[
-      [0, 4],
-      [0, 10],
+      [
+        // Creates a selection from the 5th (index 4) to the 11th
+        // (index 10) character on the first line (index 0)
+        [0, 4],
+        [0, 10],
+      ],
     ]}
   />,
 );
 ```
 
 It is often better, however, to use one of the three provided utility functions
-for selection. Though they all return a two by two array of numbers, they can be
-more descriptive than an array literal.
+for selection. Though they all return a array with one selection entry they can
```

Comment:
nit: Though they all return **an** array with one selection entry

Reply:
No longer relevant due to text changes